.github/workflows/rust.yml

```diff
@@ -28,549 +28,122 @@ jobs:
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v3
-
-      - uses: actions-rs/toolchain@v1
+      - uses: dtolnay/rust-toolchain@master
```

Comment:
@billy1624, sorry, it was mistake or expected that here is unstable rust compiler?

Reply:
@ikrivosheev thanks for double checking! This is done intentionally :)